### ISSUE TITLE
"contains" not work

### ISSUE THREAD

**Issue by Justuser3310** [SkriptLang/Skript#5432]
### Skript/Server Version

```markdown
[16:08:38 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[16:08:38 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[16:08:38 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[16:08:38 INFO]: [Skript] Server Version: git-Purpur-1858 (MC: 1.19.2)
[16:08:38 INFO]: [Skript] Skript Version: 2.6.4
[16:08:38 INFO]: [Skript] Installed Skript Addons: None
[16:08:38 INFO]: [Skript] Installed dependencies: None
```

### Bug Description

When I use this:
```
on chat:
    loop all players:
        execute console command "/me %message%"
        execute console command "/me %loop-player%"
```
Output is:
![image](https://user-images.githubusercontent.com/53183489/216979974-40d43f64-3f6e-4b65-975a-bf6c96b585b4.png)

When I add condition `message contains loop-player` then it not work.
But `message contains "_SAN5_SkeLet0n_"` work.

### Expected Behavior

Has output.

### Steps to Reproduce

`on chat:
    loop all players:
        message contains loop-player
        execute console command "/me %message%"
        execute console command "/me %loop-player%"`

### Agreement

- [x] I have read the guidelines above and affirm I am following them with this report.

**Comment by AyhamAl-Ali** [comment]
It should `loop-player's name` to compare string with string, you're comparing a string with Player
or `"%loop-player%"`

This can be improved by adding a converter for Player - String

**Comment by Moderocky** [comment]
> We will keep this opened as Enhancement to add a converter for `String - Player` to allow the use of example `message contains loop-player`

Shouldn't it be a converter of player to string?

**Comment by AyhamAl-Ali** [comment]
> Shouldn't it be a converter of player to string?

Yep, I miswrote that the second time. It should also be `CommandSender - String` to support something like `full command contains sender # console`
